tests/graphmap.rs

```diff
@@ -119,7 +119,7 @@ fn remove_node() {
     graph.remove_node(2);
 
     let neighbors: Vec<u32> = graph.neighbors(1).collect();
-    assert_eq!(neighbors, []);
+    assert_eq!(neighbors, [] as [u32; 0]);
```

Comment:
This was necessary, because the new serde_json test dependency also implemented partial_eq for those types which lead to a conflict.